src/devices/CharacterLcd/CharacterLcd.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <!--Disabling default items so samples source won't get build by the main library-->
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <LangVersion>7.2</LangVersion>
```

Comment:
NIT: Any particular reason why you picked 7.2? If not, I would just use Latest here

Reply:
Just cause that was the minbar- I can change it.